firmware/scripts/rgbconv.py

```diff
@@ -67,21 +95,35 @@ def main(*args):
     else:
         answer = None
 
-    return(answer)
+    return answer
 
 
-if __name__ == '__main__':
```

Comment:
For those type of changes (from single quotes to double quotes), it is nice to read about it [here](https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python). Basically I think we already uses single quotes for fixed small strings or constants, and double quotes to the other uses

Reply:
I think this was `black` autoformating 😵‍💫